ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -48,6 +48,7 @@
  * <p>This class is safe for use across multiple threads.
  */
 public class PendingTransactions {
+  public static final int PENDING_TX_RETENTION_PERIOD = 13;
```

Comment:
Need a unit in the name.  Seconds?  Minutes?  Hours? PENDING_TX_RETENTION_HOURS?